de.dlr.sc.virsat.model.extension.funcelectrical/src/de/dlr/sc/virsat/model/extension/funcelectrical/excel/exporter/FuncElecExporter.java

```diff
@@ -49,21 +49,26 @@
  */
 public class FuncElecExporter implements IExport {
 	
-	public FuncElecExporter(LocalDateTime localDateTime) {
-		this.localDateTime = localDateTime;
-	}
-	
-	protected LocalDateTime localDateTime;
 	private static final String DEFAULT_TEMPLATE_PATH = "/resources/ExcelExportTemplate.xlsx";
-	private ExcelExportHelper helper;
-
+	
 	private static final String[] EXPORTABLE_SEIS = {
 			ElementDefinition.class.getSimpleName(),
 			ElementConfiguration.class.getSimpleName(),
 			InterfaceTypeCollection.class.getSimpleName(),
 			ElementRealization.class.getSimpleName(),
 			ElementOccurence.class.getSimpleName()
 	};
+	
+	protected LocalDateTime localDateTime;
+	private ExcelExportHelper helper;
+	
+	public FuncElecExporter() {
+		this.localDateTime = LocalDateTime.now(); 

```

Comment:
Maybe call the other constructor instead of setting the field?

Reply:
Good idea!